src/chapters/orm-relationships/exercises.rst

```diff
@@ -0,0 +1,24 @@
+Exercises: Early Bird gets the ORM!
+===================================
+
+Now that we have gotten very familiar with our ``coding-events`` application, let's design some additional features.
+As you work on your ``coding-events`` application, you may have been inspired by `Meetup <https://www.meetup.com/>`_!
+One of the cool features that Meetup has is that we can sign up for accounts to follow the events we are most interested in and keep track of our calendar of events.
+For the exercises, answer the following questions about what your ``Person`` class would look like.
+
+.. admonition:: Note
+
+   You do not have to code anything to complete these exercises! This is mainly focused on using our design skills to add a new feature to your application.
+
+#. So we need to add a ``Person`` class to hold necessary info about users of our app. What fields and methods would this class hold?
+#. Would you need to add any additional classes to ``Person`` to make the app work?
+#. What kinds of relationships would ``Person`` have to other classes we have already created?
+
+As you dream up answers to these questions, write them down in a note or piece of paper. We are now going to write up some documentation for our app!
```

Comment:
Consistent subject edit:  "As you dream up..." should be followed by another 'you' (e.g. "Next, you are going to write up...").

Reply:
Done!